modules/openapi-generator/src/test/resources/3_0/javascript/petstore-with-fake-endpoints-models-for-testing.yaml

```diff
@@ -1272,9 +1246,6 @@ components:
       type: http
       scheme: bearer
       bearerFormat: JWT
-    http_signature_test:
-      type: http
-      scheme: signature
```

Comment:
@tiffmaelite thanks for the PR but why not still include the test anyway as it shouldn't break the client/server, right?

Reply:
can you please PM me via Slack to further discuss this PR when you've time?

Reply:
@wing328 will do somewhen tomorrow or early next week.

I also wanted to add a last argument here: what is the point of having files named `petstore-with-fake-endpoints-models-for-testing.yaml` and `petstore-with-fake-endpoints-models-for-testing-with-http-signature.yaml` if both contain the http-signature scheme?

But maybe it does not make sense to have both of them side by side, because they are quite hard to maintain (and actually one can see that their content on some endpoints diverges already for no other reason than "someone forgot about the other file")